src/webresource-handler/index.ts

```diff
@@ -277,34 +277,49 @@ export const normalizeHref = (href: string) => {
 	return href.split('?')[0];
 };
 
+const getWebResourcesKeysFromRequest = (
+	fields: string[],
+	{ values }: uriParser.ODataRequest,
+): string[] => {
+	return fields.map((field) => values[field].href);
+};
+
```

Comment:
Potentially also wbeResourceFields instead of fields? 😉
```suggestion
const getWebResourcesKeysFromRequest = (
	webResourceFields: string[],
	{ values }: uriParser.ODataRequest,
): string[] => {
	return webResourceFields.map((field) => values[field].href);
};

```